.github/workflows/zeebe-aws-os-dispatch-tests.yml

```diff
@@ -0,0 +1,106 @@
+name: AWS OpenSearch Manual Run Integration Tests
+
+on:
+  workflow_dispatch: { }
+  push: { }
```

Comment:
What is your goal, on which branches should this workflow run automatically? All branches? Just the `main` branch?

If the answer is all branches, then we should use [these triggers](https://github.com/camunda/camunda/blob/main/.github/workflows/ci.yml#L5-L10).
If the answer is just the `main` branch, then limit the existing `on.push` trigger accordingly.

Reply:
At this moment on.push added for pipeline testing. will be removed before merge and on.workflow_dispatch would be only available option